### PR TITLE
HoC 2022 - i18n strings batch #1

### PR DESCRIPTION
**Pull request by kelbyhawn**
Add new strings for Hour of Code 2022 and https://hourofcode.com homepage updates:
- HoC.com - Homepage - Hero banner - [link in Brief](https://docs.google.com/document/d/12D64T4NrCY3E8mKmvn1baL9J3b8GkyXJfUAiLSnLZeA/edit#bookmark=id.jnjiehqevshp)
- Code.org - Homepage banner 1 - HoC is coming - [link in Brief](https://docs.google.com/document/d/12D64T4NrCY3E8mKmvn1baL9J3b8GkyXJfUAiLSnLZeA/edit#bookmark=id.5wjks3tlpsrl)
- Code.org - Homepage banner 2 - Promo banner - [link in Brief](https://docs.google.com/document/d/12D64T4NrCY3E8mKmvn1baL9J3b8GkyXJfUAiLSnLZeA/edit#bookmark=id.vj07wqujnyn2)
- OpenGraph image 1 - Explore, Play, Create theme - [link in Brief](https://docs.google.com/document/d/12D64T4NrCY3E8mKmvn1baL9J3b8GkyXJfUAiLSnLZeA/edit#bookmark=id.mzm6xg7vld0c)
- HoC.com homepage updates - [Asana task](https://app.asana.com/0/0/1202772113384583/f) • [Figma link](https://www.figma.com/file/BOqojh6mhYzDd1Flu1PWoH/HoC-2022---Explore%2C-Play%2C-Create?node-id=1050%3A3254)

**Relevant info:** [Asana task](https://app.asana.com/0/1202764401306810/1203015554518693/f) • [Creative brief](https://docs.google.com/document/d/12D64T4NrCY3E8mKmvn1baL9J3b8GkyXJfUAiLSnLZeA/edit#)

**Related PRs:** https://github.com/code-dot-org/code-dot-org/pull/48187 • https://github.com/code-dot-org/code-dot-org/pull/48237 (fixes broken link) • https://github.com/code-dot-org/code-dot-org/pull/48614

### CODE REVIEW

File: pegasus/sites.v3/hourofcode.com/i18n/en.yml
Comment (kelbyhawn) [inline pegasus/sites.v3/hourofcode.com/i18n/en.yml:412]: These are two new buttons for the Nov HoC banner on code.org - [see Creative brief](https://docs.google.com/document/d/12D64T4NrCY3E8mKmvn1baL9J3b8GkyXJfUAiLSnLZeA/edit#bookmark=id.vj07wqujnyn2)

File: pegasus/sites.v3/hourofcode.com/i18n/en.yml
Comment (kelbyhawn) [inline pegasus/sites.v3/hourofcode.com/i18n/en.yml]: These `hoc_homepage…` strings are used on the light redesign of the hourofcode.com homepage - [see Asana task](https://app.asana.com/0/0/1202772113384583/f)

File: pegasus/sites.v3/hourofcode.com/i18n/en.yml
Comment (breville) [inline pegasus/sites.v3/hourofcode.com/i18n/en.yml:417]: Inconsistent: "year-round" here vs. "year round" above.

File: pegasus/sites.v3/hourofcode.com/i18n/en.yml
Comment (breville) [inline pegasus/sites.v3/hourofcode.com/i18n/en.yml]: Where is the first part of this sentence?  

We generally want to translate entire sentences at a time, in case another language can't rely on this particular ordering.

File: pegasus/sites.v3/hourofcode.com/i18n/en.yml
Comment (breville) [inline pegasus/sites.v3/hourofcode.com/i18n/en.yml]: Similar content on partial sentences.

File: pegasus/sites.v3/hourofcode.com/i18n/en.yml
Comment (bethanyaconnor) [reply pegasus/sites.v3/hourofcode.com/i18n/en.yml]: In addition, I think we generally try to use markdown instead of `a` tags. So "[read our How-to guide]({% howto_guide_url})" in this case

Comment (kelbyhawn) [reply pegasus/sites.v3/hourofcode.com/i18n/en.yml]: The first part of the sentence is a button that says "Try an activity" (`cta_try_an_activity: 'Try an activity'`) — I'll remove the "or" and make it "[Read our How-to guide]({% howto_guide_url}) to plan an event for your class."

Here it is in context w/o the "or":
Screen Shot 2022-09-20 at 12 12 41 PM

File: pegasus/sites.v3/hourofcode.com/i18n/en.yml
Comment (kelbyhawn) [reply pegasus/sites.v3/hourofcode.com/i18n/en.yml]: This will be followed by the Code.org logo:
Screen Shot 2022-09-20 at 12 14 03 PM

Comment (kelbyhawn) [reply pegasus/sites.v3/hourofcode.com/i18n/en.yml]: This isn't a definite yet either (we're waiting on Cameron's approval) …I'll make this a full sentence: `The Hour of Code is organized by [Code.org](%{codeorg_url}).` and include the logo underneath if we add this to the page.

**Review by bethanyaconnor — APPROVED** [review]

File: pegasus/sites.v3/hourofcode.com/i18n/en.yml
Comment (castro-jorge) [reply pegasus/sites.v3/hourofcode.com/i18n/en.yml]: Has Cameron approved?

**Comment by castro-jorge** [comment]
Looks good to me; thanks for the heads up Kelby!

File: pegasus/sites.v3/hourofcode.com/i18n/en.yml
Comment (kelbyhawn) [reply pegasus/sites.v3/hourofcode.com/i18n/en.yml]: Not yet, but it'll be quick to add if/when he does.

**Pull request merged**
